apps/frontend/lib/clients/oso-app/oso-app.ts

```
@@ -1992,6 +2016,16 @@ class OsoAppClient {
       .eq("org_name", orgName)
       .throwOnError();
 
+    try {
+      const hasFetch = typeof fetch === "function";
```

Why silently let this fail? Wouldn't it make more sense to have function fail/throw?

You can always call the function again (it's idempotent)

---

You're right, addressed!